deps/chakrashim/src/jsrtutils.cc

```diff
@@ -977,33 +977,46 @@ StringUtf8::~StringUtf8() {
 }
 
 JsErrorCode StringUtf8::LengthFrom(JsValueRef strRef) {
-    CHAKRA_ASSERT(_length == 0);
+  CHAKRA_ASSERT(_length == 0);
 
-    size_t len = 0;
-    IfJsErrorRet(JsCopyString(strRef, nullptr, 0, nullptr, &len));
+  size_t len = 0;
+  IfJsErrorRet(JsCopyString(strRef, nullptr, 0, nullptr, &len));
 
-    _length = len;
-    return JsNoError;
+  _length = len;
+  return JsNoError;
 }
 
 JsErrorCode StringUtf8::From(JsValueRef strRef) {
   CHAKRA_ASSERT(!_str);
 
-  IfJsErrorRet(LengthFrom(strRef));
-  size_t len = length();
+  int strLength = 0;
+  IfJsErrorRet(JsGetStringLength(strRef, &strLength));
 
-  char* buffer = reinterpret_cast<char*>(malloc(len+1));
-  CHAKRA_VERIFY(buffer != nullptr);
+  // assume string contains ascii characters only
+  _str = reinterpret_cast<char*>(malloc(strLength + 1));
```

Comment:
We should probably check the malloc return value for null and bail out like the code previously did?